doc/source/programs/nodata_handling_gdaladdo_gdal_translate.rst

```diff
@@ -1,3 +1,9 @@
+:orphan:
+
+==================================================
+Handling NoData with gdaladdo or gdal_translate
+==================================================
```

Comment:
```suggestion
.. ==================================================
.. Handling NoData with gdaladdo or gdal_translate
.. ==================================================
```

Reply:
Well, the idea was to provide the title to fix #10048.
But thinking more about it, setting the file as orphaned should be enough to fix the issue here.